### PR TITLE
Private key custom preset guides

### PR DESCRIPTION
**Pull request by fboucquez**
A small guide about setting up known private keys.

@segfaultxavi , all yours!

### CODE REVIEW

File: docs/presetGuides.md [hunk 1]
```diff
@@ -98,6 +98,42 @@ symbol-bootstrap link
...
+nodes:
+  - voting: true
+    signingPrivateKey: CA82E7ADAF7AB729A5462A1BD5AA78632390634904A64EB1BB22295E2E1A1BDD
+    vrfPrivateKey: F3C24C153783B683E40FB2671493B54480370BF4E3AB8027D4BF1293E14EB9B8
+    votingPrivateKey: EFE3F0EF0AB368B8D7AC194D52A8CCFA2D5050B80B9C76E4D2F4D4BF2CD461C1
+```
+
+Usage examples:
+- Rebuild one of your Testnet/Main nodes. 
+- Create a new node from migrated opted in accounts (included VRF)
```
Comment (segfaultxavi) [inline docs/presetGuides.md:115]:
```suggestion
- Create a new node from migrated opted in accounts (included VRF).
```

File: docs/presetGuides.md [hunk 2]
```diff
@@ -98,6 +98,42 @@ symbol-bootstrap link
...
+nodes:
+  - voting: true
+    signingPrivateKey: CA82E7ADAF7AB729A5462A1BD5AA78632390634904A64EB1BB22295E2E1A1BDD
+    vrfPrivateKey: F3C24C153783B683E40FB2671493B54480370BF4E3AB8027D4BF1293E14EB9B8
+    votingPrivateKey: EFE3F0EF0AB368B8D7AC194D52A8CCFA2D5050B80B9C76E4D2F4D4BF2CD461C1
+```
+
+Usage examples:
+- Rebuild a network with the same configuration
+- Replicate integration tests that may use specific values
```
Comment (segfaultxavi) [inline docs/presetGuides.md:133]:
```suggestion
- Rebuild a network with the same previous configuration.
- Replicate integration tests that may use specific values.
```

File: docs/presetGuides.md [hunk 3]
```diff
@@ -98,6 +98,42 @@ symbol-bootstrap link
...
+
+Key your generated `addresses.yml` and `preset.yml` in the target folder privately!
```
Comment (segfaultxavi) [inline docs/presetGuides.md:135]:
```suggestion
Keep the generated `addresses.yml` and `preset.yml` in the target folder privately!
```

File: docs/presetGuides.md [hunk 4]
```diff
@@ -98,6 +98,42 @@ symbol-bootstrap link
...
+
+If you want to replicate the nemesis block and generation hash of your private bootstrap network, you can specify the nemesis keys:
+
+```yaml
+nodes:
+  - voting: true
+    signingPrivateKey: CA82E7ADAF7AB729A5462A1BD5AA78632390634904A64EB1BB22295E2E1A1BDD
+    vrfPrivateKey: F3C24C153783B683E40FB2671493B54480370BF4E3AB8027D4BF1293E14EB9B8
+    votingPrivateKey: EFE3F0EF0AB368B8D7AC194D52A8CCFA2D5050B80B9C76E4D2F4D4BF2CD461C1
+```
```
Comment (segfaultxavi) [inline docs/presetGuides.md:129]: This yaml snippet looks exactly like the previous one. Copy&Paste error?

**Review by segfaultxavi — CHANGES_REQUESTED** [review]
Nice :)

File: docs/presetGuides.md (hunk 4 above)
Comment (fboucquez) [reply docs/presetGuides.md]: Ups, pushed the network namesis example

**Review by segfaultxavi — APPROVED** [review]

**Pull request merged**